spec/feature_flagger/configuration_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'spec_helper'
+
+module FeatureFlagger
+  RSpec.describe Configuration do
+    context 'accessors methods' do
+      subject(:configuration) { described_class.new }
+
+      it { expect(configuration).to respond_to(:storage) }
+      it { expect(configuration).to respond_to(:storage=) }
+      it { expect(configuration).to respond_to(:yaml_filepath) }
+      it { expect(configuration).to respond_to(:info) }
+    end
```

Comment:
https://github.com/ResultadosDigitais/feature_flagger/pull/17/files#diff-1471e2e6e97c2400b4b079ebf2df4416R25 Here is testing the default value for Configuration, but testing the high level in FeatureFlagger.

It's better to move this kind os specs to low level and test in Configuration and remove this specs from FeatureFlagger ?